src/styles/partials/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$padding: 1rem;
+$margin: 1rem;
+$font-size: 16px;
+$font-width-normal: 400;
+$font-width-medium: 500;
+$font-width-thick: 700;
+$full-width: 100%;
+$line-height: 26px;
+$image-width: 30px;
```

Comment:
`$vertical-overflow-scrollable` and `$border-bottom-unset` seem rather specific. Otherwise LGTM 👍